### ISSUE TITLE
Resizable things

### ISSUE THREAD

**Issue by jbpagliuco** [lsst-camera-visualization/frontend-old#63]
Boxes, viewers, and terminal should be resizable. I believe there's a jquery ui plugin for this

**Comment by slm23** [comment]
If you work this out it would also be great if the ffviewer started off with a square aspect ratio for the image part of the window.  I think the whole window is square which makes the image part rectangular.  Our CCDs and rafts are square so that's a better default shape.

**Comment by weiren2** [comment]
@slm23 I tried a quick change in the css. Would this ffviewer below looks better? 
![image](https://cloud.githubusercontent.com/assets/6765243/17644502/a0a3a944-614d-11e6-8c49-446dd6b842c3.png)

Edit: I found that a second viewer would overlap with the terminal in this case. This might not be a problem since user can also drag the terminal now (on latest master branch). Maybe there are other better ways to handle the UI.
![image](https://cloud.githubusercontent.com/assets/6765243/17644530/dd834508-614e-11e6-952a-d4f395e850d8.png)
